src/app/modules/submission/submission.errors.ts

```diff
@@ -167,12 +167,26 @@ export class InvalidFileKeyError extends ApplicationError {
   }
 }
 
+export class GuardDutyInvalidFileKeyError extends ApplicationError {
```

Comment:
/nit i see a couple of casing for `GuardDuty` vs `Guardduty` in this PR. Would prefer `GuardDuty` if we're renaming the new additions.

Reply:
Okay, will change the casing to match `GuardDuty`